src/lib/db/feature-strategy-store.ts

```diff
@@ -317,6 +318,27 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         };
     }
 
+    private isUnseenTag(
+        feature: Record<string, any>,
+        row: Record<string, any>,
+    ): boolean {
+        return (
+            row.tag_type &&
+            row.tag_value &&
+            !feature.tags.find(
```

Comment:
Here also `!some`

Reply:
Thanks, mostly artifacts left from copy/pasting from other places. Addressed in https://github.com/Unleash/unleash/pull/2396/commits/22eff75b2cdf902b20a1da8df2d94a5259330046